analysis/crystal/crystal.go

```diff
@@ -0,0 +1,85 @@
+package crystal
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+// Host is the base URL for the analysseur API.
```

Comment:
Sure, that sounds good! I'll at least make the changes to the `crystal-analyzer` flag in here.